tests/envs/test_registration.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
+import pytest
+
 import gym
-from gym import error, envs
+from gym import envs, error
```

Comment:
Are these formatting changes necessary to pass CI?

Reply:
No no, they're not, I just used [isort](https://pycqa.github.io/isort/) so that imports are standardized (kind of `black` but for imports statements), but I can revert that if you prefer.